Sources/AblyChat/EmitsDiscontinuities.swift

```diff
@@ -1,5 +1,5 @@
 import Ably
 
 public protocol EmitsDiscontinuities {
-    func subscribeToDiscontinuities() -> Subscription<ARTErrorInfo>
```

Comment:
Can you please make sure that the commit messages explain any changes that you needed to make to the public API?

Reply:
This particular explanation is similar to https://github.com/ably-labs/ably-chat-swift/pull/34/commits/64e400dddbe0e7b3b895811c99dd9bedb68fe5b9
I'm going to wrap everything into one commit after this PR approved with all the explanations for the API changes.

Reply:
https://github.com/ably-labs/ably-chat-swift/pull/34/commits/961e17b2c9b6668d691dcf595365cd1bb03a7612